### DISCUSSION TITLE
react-table on React Native?

### DISCUSSION THREAD

**Discussion by James-Reed-cognito** [TanStack/table#2868] [Q&A]
Hey there!

I'm just looking around at this library and it looks great at first look for a webapp! Does it work on React Native?

**Comment by frofro1979** [comment]
react-table-6 is the native verion. I'd use the hooks version if I were you though

**Reply by skychavda** [reply]
Can you provide me with some examples for this?

**Comment by James-Reed-cognito** [comment]
@frofro1979 the 'hooks' version? I thought this library was all about hooks?
